### PR TITLE
access to id, service and subservice in jexl context for functional test

### PR DESCRIPTION
**Pull request by AlvaroVega**
fix for https://github.com/telefonicaid/iotagent-node-lib/pull/1523

### CODE REVIEW

File: test/functional/testCases.js [hunk 1]
```diff
@@ -1785,7 +1784,7 @@ const testCases = [
                                 object_id: 'a',
                                 name: 'attr_a',
                                 type: 'Text',
-                                expression: 'a+":"+service+subservice+id+type'
```
Comment (AlvaroVega) [inline test/functional/testCases.js:1788]: groupToIoTAConfigType is not providen deviceId

Maybe  testUtils.testCase should do that

**Comment by fgalan** [comment]
I think it would be wise to use same literals than the one used at CB (https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#macro-substitution)

> * `id`: for the `id` of the entity
> * `type`: for the `type` of the entity
> * `service`: for the service (i.e. `fiware-service` header value) in the
>   update request triggering the subscription.
> * `servicePath`: for the service path (i.e. `fiware-servicepath` header value) in the
>   update request triggering the subscription.
> * `authToken: for the authorization token (i.e. `x-auth-token` header value) in the
>   update request triggering the subscription.
> * All the attributes in the entity triggering the notification (included in the update triggering the notification or not)

Is the PR working that way?

**Comment by AlvaroVega** [comment]
> Is the PR working that way?

Not. This is just to align functional tests with implemented feature in iotagent-node-lib, specially test about jexl context

**Comment by fgalan** [comment]
> > Is the PR working that way?
> 
> Not. This is just to align functional tests with implemented feature in iotagent-node-lib, specially test about jexl context

CB uses `servicePath`. IOTA uses `subservice`. They are different but probably is not a good idea to change any of them to align, due to potential backward compatibility problems and changes in the IOTA DB model.

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**